ompi/mpi/c/Makefile.am

```diff
@@ -437,7 +437,8 @@ libmpi_c_mpi_la_SOURCES = \
         win_wait.c
 
 
-if OMPI_ENABLE_MPI1_COMPAT
```

Comment:
How about putting these files just back up (alphabetically) in the main list of files?

Also, this specific commit was not a cherry pick, right?  Can you please put a comment about that in the commit message? (i.e., why this is not a cherry pick: because on master, we are no longer building those symbols by default, but on v4.0 we *are* still building those symbols by default, yadda yadda yadda)

Reply:
I kinda liked having them separated out at the bottom to help indicate that they're going away.

I will rework the commit message to indicate why we want/need this v4.0.x specific patch (instead of a cherry-pick from master).